source/import-export.txt

```diff
@@ -95,7 +130,7 @@ Export Data from a Collection
 **JSON** or **CSV** file. If you specify a query in the
 :ref:`query bar <compass-query-bar>` prior to export, Compass can
 optionally only export documents which match the specified query. If no
-query is specified, Compass exports the entire collection.
+query is specified, |compass-short| exports the entire collection.
```

Comment:
or if you check the "Export Full Collection" toggle

Reply:
Thought about this a bit, I think just dropping this sentence entirely works better, since we explain the Export Full Collection toggle in the stepped procedure below.